controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -668,6 +674,122 @@ func TestOverallStatusCondition(t *testing.T) {
+
+		// when
+		res, err := reconciler.Reconcile(context.TODO(), req)
+
+		// then
+		require.NoError(t, err)
+		assert.Equal(t, requeueResult, res)
+		AssertThatMemberStatus(t, req.Namespace, requestName, fakeClient).
+			HasCondition(ComponentsReady()). // all components are ready
+			HasMemberOperatorConditions(ConditionReady(toolchainv1alpha1.ToolchainStatusDeploymentReadyReason)).
+			HasMemberOperatorRevisionCheckConditions(ConditionReady(toolchainv1alpha1.ToolchainStatusDeploymentUpToDateReason)).
+			HasMemoryUsage(OfNodeRole("master", 33), OfNodeRole("worker", 25)).
+			HasRoutes("https://console.member-cluster/console/", "https://codeready-codeready-workspaces-operator.member-cluster/che/", routesAvailable())
+	})
+
+	t.Run("deployment version check is disabled", func(t *testing.T) {
+		t.Run("when environment is not prod", func(t *testing.T) {
+			// given
+			// we set dev as environment
+			config := commonconfig.NewMemberOperatorConfigWithReset(t, testconfig.MemberEnvironment("dev"), testconfig.MemberStatus().GitHubSecretRef("github").GitHubSecretAccessTokenKey("accessToken"))
+			requestName := defaultMemberStatusName
+			memberOperatorDeployment := newMemberDeploymentWithConditions(status.DeploymentAvailableCondition(), status.DeploymentProgressingCondition())
+			memberStatus := newMemberStatus()
+			getHostClusterFunc := newGetHostClusterReady
+			// we have a secret that contains the access token for GitHub authenticated APIs
+			githubSecret := test.CreateSecret("github", test.MemberOperatorNs, map[string][]byte{
+				"accessToken": []byte("abcd1234"),
+			})
+			commitTimeStamp := time.Now().Add(-time.Hour * 1)
+			reconciler, req, fakeClient := prepareReconcile(t, requestName, getHostClusterFunc, allNamespacesCl, mockLastGitHubAPICall, test.MockGitHubClientForRepositoryCommits(buildCommitSHA, commitTimeStamp), append(nodeAndMetrics, memberOperatorDeployment, memberStatus, config, githubSecret)...) // let's pass the build commit
+
+			// when
+			res, err := reconciler.Reconcile(context.TODO(), req)
+
+			// then
+			require.NoError(t, err)
+			assert.Equal(t, requeueResult, res)
+			AssertThatMemberStatus(t, req.Namespace, requestName, fakeClient).
+				HasCondition(ComponentsReady()).                                                                     // all components are ready
+				HasMemberOperatorConditions(ConditionReady(toolchainv1alpha1.ToolchainStatusDeploymentReadyReason)). // we have only one condition for the deployment status, no condition for the version match
+				HasMemoryUsage(OfNodeRole("master", 33), OfNodeRole("worker", 25)).
+				HasRoutes("https://console.member-cluster/console/", "https://codeready-codeready-workspaces-operator.member-cluster/che/", routesAvailable())
```

Comment:
I can be blind, but I don't see the assertion of the revision check condition something like that:
```go
HasMemberOperatorRevisionCheckConditions(ConditionReady(toolchainv1alpha1.ToolchainStatusDeploymentRevisionCheckDisabledReason ...
```

Reply:
I are not blind, I forgot to add it back. Since we went from 2 conditions to 1 conditions and now back to 2 conditions , I forgot to update the tests 🤦‍♂️ . Good catch again!

Done in [78a30e1](https://github.com/codeready-toolchain/member-operator/pull/446/commits/78a30e10315b864af392e7a11c435de06468920a) and [daee1ad](https://github.com/codeready-toolchain/member-operator/pull/446/commits/daee1ad8962b8530c1da5e096e0f3ac12a8b7a1d)